shared/provision/paper-key/container.tsx

```diff
@@ -30,8 +34,11 @@ export default Container.connect(
       error: stateProps.error,
       hint: stateProps.hint,
       onBack:
-        loggedInAccounts.length > 0 ? () => dispatchProps._onLogIn(loggedInAccounts[0] || '') : undefined,
+        loggedInAccounts.length > 0
+          ? () => dispatchProps._onLogIn(loggedInAccounts[0] || '')
+          : dispatchProps._onBack,
```

Comment:
Have you tested this case?

Reply:
Which one? The user switching case wasn't changed, I'd rather not mess around with it as it's out of scope of the rework.

Reply:
The case where you're logging in on a fresh install with a paper key and hit "back" on that screen. I think it should take you back to join-or-login

Reply:
Yeah, it works.